Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Jacob Hoffman-Andrews <github@hoffman-andrews.com>"]
 description = "C-to-rustls bindings"
 edition = "2018"
 links = "rustls_ffi"
+rust-version = "1.52.0"
```

Comment:
A little curious that this is named "rust-version" and not "minimum-rust-version" or similar... if I was reading this I might assume it will force my compiler to 1.52.0 or only work with that version or something.